ballerina/Module.md

```diff
@@ -58,8 +58,8 @@ service kafka:Service on kafkaListener {
         // commits the offsets manually
         kafka:Error? commitResult = caller->commit();
 
-        if (commitResult is error) {
-            io:println("Error occurred while committing the offsets for the consumer ", 'error = commitResult);
+        if (commitResult is kafka:Error) {
```

Comment:
We don't want the parentheses here, right?

Reply:
Removed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/377/commits/19a48dd178c2d4bfa8fdba5298242c8dfaa7e695